core/cas-server-core-webflow-api/src/main/java/org/apereo/cas/web/flow/actions/ClearWebflowCredentialAction.java

```diff
@@ -13,7 +13,10 @@
 
 /**
  * This is {@link ClearWebflowCredentialAction} invoked ONLY as an exit-action for non-interactive authn flows.
- *
+ * Don't clear credentials when "success" occurs which leads to STATE_ID_CREATE_TICKET_GRANTING_TICKET but
```

Comment:
try to use Javadoc linking for code elements; makes future refactorings much much easier.